src/components/Home.jsx

```diff
@@ -0,0 +1,59 @@
+import { useSelector, useDispatch } from 'react-redux';
+import { useEffect } from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faGear } from '@fortawesome/free-solid-svg-icons';
+import { fetchSectors } from '../API/dataAPI';
+import Sector from './companySector';
+import style from '../styles/Sectors.module.css';
+
+function Sectors() {
+  const { sectors } = useSelector((state) => state.sectors);
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(fetchSectors());
+  }, [dispatch]);
+
+  const isLight = [0, 3, 4, 7, 8];
+  const { sector, sectorsDiv } = style;
+
+  return (
+    <section className="animated">
+      <header>
+        <div />
+        <div>Newton App</div>
+        <div><FontAwesomeIcon icon={faGear} /></div>
+      </header>
+      <div className={sector}>
+        <section className={sectorsDiv}>
+          {
+            sectors.map((sector, index) => {
+              let className = '';
+
+              if (index < 10 || (index / 10) % 2 === 0) {
+                className = isLight.includes(index % 10)
+                  ? 'is-light'
+                  : 'is-dark';
+              } else {
+                className = isLight.includes(index % 10)
+                  ? 'is-dark'
+                  : 'is-light';
+              }
+
+              return (
+                <Sector
+                  key={sector.sector}
+                  sector={sector.sector}
+                  change={sector.changesPercentage}
+                  className={className}
+                />
+              );
+            })
+          }
+        </section>
```

Comment:
- [ ] Nice work implementing the cards on the home page :clap: However I get an error when I click on the first sector (Basic Materials) :point_down: . Kindly check it 
![Screenshot from 2023-09-07 06-31-47](https://github.com/mohammadnewton/react-capstone/assets/91142432/fbfde7c4-b053-457e-852d-2a459c026a01)